src/test/resources/org/datadog/jenkins/plugins/datadog/listeners/testPipelineStageTags.txt

```diff
@@ -0,0 +1,46 @@
+pipeline {
+    agent {
+      label 'built-in'
+    }
+    stages {
+        stage('Stage 1') {
+            steps {
+                echo 'Stage 1'
+            }
+        }
+
+        stage('Stage 2') {
+            options {
+                datadog(tags: ["outer_tag:value"])
+            }
```

Comment:
Is it possible to have (and if so, what happens if):
- `tags` at root level and `tags` at the stage level at the same time?
- `tags` at root level with the same key than the `tags` at the stage level at the same time?

Reply:
> tags at root level and tags at the stage level at the same time?
yes, this is possible

> tags at root level with the same key than the tags at the stage level at the same time?
if the same tag is defined at multiple levels (root, stage, "substage"), the innermost value will be used. I actually had to fix this